src/Header.js

```diff
@@ -4,13 +4,21 @@ export default class Header extends Component {
   render() {
     return (
       <div>
-            {/* <img src={this.props.logoText} /> */}
         <header>
-          <img className="logo" src="http://www.placekitten.com/100/100" alt="Pokemon" />
+          <img className="logo" src="https://i.ebayimg.com/images/g/8bMAAOSwxp9W-TVs/s-l400.jpg" alt="Pokemon" />
           <h1>Pokedex</h1>
-        </header>
         
-      </div>
+       
+
+          <form>
+              <label for="search">Search: </label>
```

Comment:
anytime you use lables in react dont use for= use forHTML=

this will get rid of your error!